dune-project

```diff
@@ -1,2 +1,34 @@
-(lang dune 1.0)
+(lang dune 2.0)
+
 (name xenstore)
+
+(generate_opam_files true)
+
+(source
+ (github mirage/ocaml-xenstore))
+
+(license "LGPL-2.1-only WITH OCaml-LGPL-linking-exception")
+
+(authors
+ "Vincent Hanquez"
+ "Thomas Gazagnaire"
+ "Dave Scott"
+ "Anil Madhavapeddy"
+ "Vincent Bernardoff")
+
+(maintainers "Hannes Mehnert <hannes@mehnert.org>")
+
+(package
+ (name xenstore)
+ (synopsis "Xenstore protocol in pure OCam")
+ (description
+  "Implementation of the xenstore protocol in OCaml. This is an\n  alternative implementation to the implementation in the Xen project.")
+ (depends
+  (dune
+   (>= "2.0"))
```

Comment:
Dune recommends to leave a dependency on dune out: it'll automatically add a dependency based on the 'lang' version declared in dune-project.

Reply:
See https://dune.readthedocs.io/en/stable/howto/opam-file-generation.html#general-notes-and-tips